### ISSUE TITLE
de-android-ify com.joelapenna.foursquare

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#121]
```
com.joelapenna.foursquare doesn't work with raw java, fix that.
```

Original issue reported on code.google.com by `jlapenna` on 3 Nov 2009 at 12:28

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 5e786034de.

Move fromAndroidLocation to foursquared package.
```

Original comment by `jlapenna` on 13 Nov 2009 at 5:28

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision c6a0a26767.
```

Original comment by `jlapenna` on 14 Nov 2009 at 4:36
- Changed state: **Fixed**
